src/LiveQuery/ParseWebSocketServer.js

```diff
@@ -14,17 +14,23 @@ export class ParseWebSocketServer {
       logger.info('Parse LiveQuery Server started running');
     };
     wss.onConnection = ws => {
+      ws.waitingForPong = true;
```

Comment:
This should be inverted now. `this.waitingForPong = false` initially. It should only be waiting for pong after `ws.ping();` and until `ws.on('pong', ...)`.

Reply:
I'm sorry, I think just need to invert or change the variable name, and if change the name, don't need to invert the logic.
it's done now.
Thank you for your patience and kindness

Reply:
Thanks for your work! Looks perfect now